components/FilterBar/styles.ts

```diff
@@ -0,0 +1,32 @@
+import { CSSProperties } from 'react';
+
```

Comment:
use **styled components** not CSSProperties. Check Neha's recent styling sprint -- she correctly used styled components in hers.